packages/react-console/src/components/AccessConsoles/AccessConsoles.tsx

```diff
@@ -30,7 +30,7 @@ export interface AccessConsolesProps {
   children?: React.ReactElement[] | React.ReactNode;
   /** Placeholder text for the console selection */
   textSelectConsoleType?: string;
-  /** The value for the Serial Console option */
+  /** The value for the Serial Console option. This can be overriden by the type property of the child component */
```

Comment:
Can there only be multiples of the serial component? If the others can have multiple instances as well, then I think we should either add a general comment explaining how to override the option value for any of them or add this comment to each of them.